### DISCUSSION TITLE
06 18 데모 대비 소스 저장소

### DISCUSSION THREAD

**Discussion by GleamingStar** [d-h-k/issue-tracker#76] [Ideas]
댓글 ㄱㄱㄱㄱ

**Comment by GleamingStar** [comment]
- 로그인
![GIF 2021-06-18 오후 2-30-03](https://user-images.githubusercontent.com/70461368/122512350-b1d75e80-d043-11eb-9b7d-a8f3260b4e40.gif)
- 이슈리스트
![GIF 2021-06-18 오후 2-31-15](https://user-images.githubusercontent.com/70461368/122512402-c1ef3e00-d043-11eb-80f4-4c9404b34c03.gif)
- 이슈디테일
![GIF 2021-06-18 오후 2-35-22](https://user-images.githubusercontent.com/70461368/122512475-dd5a4900-d043-11eb-986d-7e93ef063264.gif)
- 라벨리스트
![GIF 2021-06-18 오후 2-33-49](https://user-images.githubusercontent.com/70461368/122512424-cb78a600-d043-11eb-9cdf-8ff153de7d50.gif)

**Comment by eeeesong** [comment]
# iOS 구현 화면 소개

## 🔐 소셜로그인
- 깃허브 & 애플 로그인 구현
- 유저 로그인 정보 키체인 저장

### Github 로그인
![gitLogin](https://user-images.githubusercontent.com/52390975/122518751-91f86880-d04c-11eb-917a-54d984abd848.gif)
- 백엔드 연동하여 jwt 통신으로 API 통신 가능 로그인/로그아웃 가능
### Apple 로그인
![apple login](https://user-images.githubusercontent.com/72188416/122516705-0e3d7c80-d04a-11eb-811e-77a3c14c8353.gif)
- 백엔드 연동 미구현으로 애플 로그인으로 서비스를 이용할 수는 없음

## 🏷 레이블
- 레이블 조회 및 추가 & 수정 & 삭제

### 레이블 조회 및 추가

![new label ](https://user-images.githubusercontent.com/72188416/122516925-55c40880-d04a-11eb-9c3f-d348b221e320.gif)

### 기존 레이블 수정
![edit label3](https://user-images.githubusercontent.com/72188416/122518322-04b51400-d04c-11eb-8144-3a2cef69c563.gif)

### 레이블 삭제
![delete label](https://user-images.githubusercontent.com/72188416/122516942-5a88bc80-d04a-11eb-951b-9cbfed5ab552.gif)

## 🗿마일스톤
- 마일스톤 조회

### 마일스톤 조회가능
![Simulator Screen Recording - iPhone 12 - 2021-06-18 at 15 52 47](https://user-images.githubusercontent.com/52390975/122519298-45615d00-d04d-11eb-8677-4fdef11ebd35.gif)

**Reply by JacksonPk** [reply]
오 굳굳!!!

**Comment by Jiwon-JJW** [comment]
* DB 설계 이미지
![Untitled](https://user-images.githubusercontent.com/69128652/122521572-f36e0680-d04f-11eb-9b84-df8b3ed0e36b.png)
